ui/src/treeDataProvider/AzureParentNode.ts

```diff
@@ -137,44 +146,42 @@ export class AzureParentNode<T extends IAzureParentTreeItem = IAzureParentTreeIt
             this._cachedChildren = [];
         }
 
-        const sortCallback: (n1: IAzureNode, n2: IAzureNode) => number =
-            this.treeItem.compareChildren
-                ? this.treeItem.compareChildren
-                : (n1: AzureNode, n2: AzureNode): number => n1.treeItem.label.localeCompare(n2.treeItem.label);
+        const sortCallback: (n1: AzureTreeItem<T>, n2: AzureTreeItem<T>) => number =
```

Comment:
n1 and n2, huh? Not ti? 😛

Reply:
If you wanted to convince me you're actually reviewing this whole PR - you've succeeded haha